### PR TITLE
Indicate default quota

### LINKED ISSUE #11393: [Enhancement] Indicate users default quota
In the administrators user list, the quota for each user is displayed. At the moment, it is not apparent which users have the default quota and which have the same quota defined manually, as only the number is displayed.

When changing a users quota to default, it reads `Default quota`. However, after a page reload, only the number is displayed again.

It would be nice to have a reading such as `10 GB (default)` or similar instead of only `10 GB`.

### PR DESCRIPTION
**Pull request by AasthaGupta**
screenshot from 2018-10-28 03-08-22

Fixes #11393

### CODE REVIEW

**Comment by violoncelloch** [comment]
PR #12081 targets the same issue

**Comment by daita** [comment]
Thanks for your comtribution, please sign your work when commiting!

https://github.com/nextcloud/appstore/blob/master/CONTRIBUTING.md#sign-your-work

**Comment by AasthaGupta** [comment]
Thanks @daita .Signed.

File: .htaccess [hunk 1]
```diff
@@ -80,3 +80,7 @@ Options -Indexes
 <IfModule pagespeed_module>
   ModPagespeed Off
 </IfModule>
+#### DO NOT CHANGE ANYTHING ABOVE THIS LINE ####
+
+ErrorDocument 403 /server/
```
Comment (danielkesselberg) [inline .htaccess]: Could you revert this?

**Comment by danielkesselberg** [comment]
> Regenerating core/vendor/core.js
The merged vendor file is not up to date
Please run: php build/mergejs.php
And commit the result

File: core/js/mimetypelist.js [hunk 2]
```diff
@@ -100,8 +100,7 @@ OC.MimeTypeList={
     "text/x-ldif": "text/code",
     "text/x-python": "text/code",
     "text/x-shellscript": "text/code",
-    "web": "text/code",
-    "application/internet-shortcut": "link"
+    "web": "text/code"
```
Comment (juliushaertl) [inline core/js/mimetypelist.js]: I guess this change also doesn't belong here :wink:

**Comment by juliushaertl** [comment]
- [ ] There seems to be a bug if the default quota is set to unlimited. In that case the UI always shows 1GB for users with default quota.

**Comment by danielkesselberg** [comment]
Ref #12159

**Comment by ChristophWurst** [comment]
-> integrated the review feedback in a cherry-picked version: https://github.com/nextcloud/server/pull/13909

**Pull request closed without merging**